packages/web-ui/src/Text/Text.docs.mdx

```diff
@@ -33,7 +33,7 @@ overflow).
 
 ### Contextual colour
 
-When inside a `Box` component that specifies a `backgroundColor` which is the
+When inside a `Box` component that specifies a `background` which is the
```

Comment:
backgroundColor is more specific and explicit than the CSS shorthand, maybe it's better to keep it that way?

Reply:
The prop is less about setting the background colour CSS value and more about setting the background context for child elements.